src/base/traverseFunctions.js

```diff
@@ -402,64 +427,62 @@ export function markVisibleTiles( tile, renderer ) {
 }
 
 // Final traverse to toggle tile visibility.
-export function toggleTiles( tile, renderer ) {
+export function toggleTiles( root, renderer ) {
```

Comment:
This change will result in a different and more complex traversal than the previous one. The current implementation in master [conditionally traverses the children](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/master/src/base/traverseFunctions.js#L408) so it will stop early in the hierarchy. These tile traversal functions are complex and can be a bit difficult to test and benchmark - I'm happy to make changes to them but it might be best to keep them in a dedicated PR so they're easier to analyze and test individually.

The stack-based `traverseSet` update looks like a good change, though!